inc/commontreedropdown.class.php

```diff
@@ -480,6 +480,12 @@ function showChildren() {
       $nb            = count($fields);
       $entity_assign = $this->isEntityAssign();
 
+      foreach ($fields as &$field) {
```

Comment:
Usage of `&` referencing operator as side effects. Displayed name in headers for last column is not the good one.

As the whole method is made to use fields having the `list` flag to true, we can do:
```diff
diff --git a/inc/commontreedropdown.class.php b/inc/commontreedropdown.class.php
index 40b080df0e..fd24ec99cb 100644
--- a/inc/commontreedropdown.class.php
+++ b/inc/commontreedropdown.class.php
@@ -476,7 +476,12 @@ abstract class CommonTreeDropdown extends CommonDropdown {
 
       $ID            = $this->getID();
       $this->check($ID, READ);
-      $fields        = $this->getAdditionalFields();
+      $fields = array_filter(
+         $this->getAdditionalFields(),
+         function ($field) {
+            return isset($field['list']) && $field['list'];
+         }
+      );
       $nb            = count($fields);
       $entity_assign = $this->isEntityAssign();
```
We could also remove following `if ($field['list']) {` tests as they would be useless.